packages/firebase_storage/ios/Classes/FirebaseStoragePlugin.m

```diff
@@ -88,4 +90,19 @@ - (void)getData:(FlutterMethodCall *)call result:(FlutterResult)result {
             }];
 }
 
+- (void)deleteData:(FlutterMethodCall *)call result:(FlutterResult)result {
+  NSString *path = call.arguments[@"path"];
+  FIRStorageReference *ref = [[FIRStorage storage].reference child:path];
+  [ref deleteWithCompletion:^(NSError *error){
+  if (error != nil) {
+    result(error.flutterError);
```

Comment:
I simply passed error as it was done before, not sure if some additional data should be passed (something like `NO` value might be good)